### PR TITLE
Add SubjectWorkflowStatus object (in progress)

### PR DESCRIPTION
**Pull request by mwalmsley**
API response reads correctly out-of-the-box, which is cool.

.where() only appears to work with id= arg - any idea what's missing, @adammcmaster ?

*Not tested!* (let's see if this PR triggers a build...)

### CODE REVIEW

**Comment by camallen** [comment]
> API response reads correctly out-of-the-box, which is cool.
> 
> .where() only appears to work with id= arg - any idea what's missing, @adammcmaster ?
> 
> _Not tested!_ (let's see if this PR triggers a build...)

OOI - what where clause did you try? you can filter on the `id`, `subject_id` and `workflow_id` API side.

**Comment by mwalmsley** [comment]
I tried with `classifications_count` and `retirement_reason`, but I guess those aren't implemented on the API side. `id`,`subject_id` and `workflow_id` all work - thanks Cam.

File: panoptes_client/subject_workflow_status.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+from panoptes_client.panoptes import PanoptesObject
+
+
+class SubjectWorkflowStatus(PanoptesObject):
+    """
+    Retrieve SubjectWorkflowStatus responses from Panoptes i.e. the retirement status (current state, retirement date, retirement reason) of a subject/workflow pair.
```
[the remark this answers is not in the document]
Comment (adammcmaster) [reply panoptes_client/subject_workflow_status.py]:
```suggestion
    Retrieve SubjectWorkflowStatus responses from Panoptes i.e. the retirement
    status (current state, retirement date, retirement reason) of a
    subject/workflow pair.
```

File: panoptes_client/subject_workflow_status.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+    Example use:
+
+    Get the status of a given subject:
+        subject_workflow_status = next(SubjectWorkflowStatus.where(subject_id='30089908'))
```
[the remark this answers is not in the document]
Comment (adammcmaster) [reply panoptes_client/subject_workflow_status.py]:
```suggestion
        subject_workflow_status = next(
            SubjectWorkflowStatus.where(subject_id='30089908')
        )
```

File: panoptes_client/subject_workflow_status.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+    The .where(kwargs) method works with
```
[the remark this answers is not in the document]
Comment (adammcmaster) [reply panoptes_client/subject_workflow_status.py]:
```suggestion
    The .where(kwargs) method works with:
```

File: panoptes_client/subject_workflow_status.py [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+    """
+    Retrieve SubjectWorkflowStatus responses from Panoptes i.e. the retirement status (current state, retirement date, retirement reason) of a subject/workflow pair.
+
+    Example use:
+
+    Get the status of a given subject:
+        subject_workflow_status = next(SubjectWorkflowStatus.where(subject_id='30089908'))
+
+    The .where(kwargs) method works with 
+    - id (i.e. the id of the SubjectWorkflowStatus, which is *not* the same as the subject_id)
```
[the remark this answers is not in the document]
Comment (adammcmaster) [reply panoptes_client/subject_workflow_status.py]:
```suggestion
    - id (i.e. the id of the SubjectWorkflowStatus, which is *not* the same as
      the subject_id)
```

File: panoptes_client/subject_workflow_status.py [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+    - subject_id
+    - workflow_id
+
+    Remember that one subject may be classified on many workflows, and hence may have many SubjectWorkflowStatus' (one per subject/workflow pair).
```
[the remark this answers is not in the document]
Comment (adammcmaster) [reply panoptes_client/subject_workflow_status.py]:
```suggestion
    Remember that one subject may be classified on many workflows, and hence
    may have many SubjectWorkflowStatus' (one per subject/workflow pair).
```

**Comment by mwalmsley** [comment]
Lines wrapped, Hound fed.

**Pull request merged**